.drone.yml

```diff
@@ -146,12 +146,12 @@ steps:
     AWS_SECRET_ACCESS_KEY:
       from_secret: put-drone-aws-sk
   commands:
-  - bash -i ./ci/nctl_upgrade.sh
+  - bash -c ./ci/nctl_upgrade.sh
 
 - name: check CPU features
   <<: *buildenv
   commands:
-  - bash -i ./ci/check_cpu_features.sh  
+  - bash -i ./ci/check_cpu_features.sh
```

Comment:
Any reason to not run this as non-interactive too?

Reply:
I think it failed if I remember right, so I thought this could get left as is since it will only create minimal pollution of the logs.